package.json

```diff
@@ -26,7 +26,7 @@
     "redux": "^4.2.1",
     "redux-devtools-extension": "^2.13.9",
     "sass-mediaqueries": "^1.6.1",
-    "shared-react-components": "github:precision-sustainable-ag/shared-react-components#feature/figmaTextfield",
+    "shared-react-components": "github:precision-sustainable-ag/shared-react-components#feature/figmaAccordion",
```

Comment:
this line is incorrect it should always be     
`"shared-react-components": "github:precision-sustainable-ag/shared-react-components",`

Reply:
This is just for reviewing, after the code is reviewed and the changes are merged in the shared repo it should be updated to the main branch